src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <h1>Krasnoshchok's</h1>
+    <form
+      onsubmit="onSubmit()"
+      autocomplete="off"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset>
+        <legend>Personal Information</legend>
+        <label for="personalSurname">
+          Surname :
+          <input type="text"
+          name="surname"
+          id="personalSurname" >
+        </label>
+        <label for="personalName">
+          Name :
+          <input type="text"
+          name="name"
+          id="personalName" >
+        </label>
+        <label for="personalAge">
+          How old are you?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            max="100"
+            id="personalAge"
+          >
+        </label>
+        <label for="personalDate">
+          Full date of birth :
+          <input type="date"
+          name="dateOfBirth"
+          id="personalDate" >
+        </label>
+        <label for="personalAccept">
+          I agree the term of the agreement
+          <input
+            type="checkbox"
+            name="termAccept"
+            id="personalAccept"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="registrationEmail">
+          Email :
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="email"
+            id="registrationEmail"
+          >
+        </label>
+        <label for="registrationPass">
+          Password :
+          <input
+            type="password"
+            minlength="5"
+            maxlength="32"
+            placeholder="6 didits"
+            pattern="\d+"
+            name="password"
+            id="registrationPass"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div>
+          Do you love cats?
+          <label for="factCats1">
+            <input type="radio"
+            value="yes"
+            name="loveCats"
+            id="factCats1">
```

Comment:
```suggestion
            <input 
              type="radio"
              value="yes"
              name="loveCats"
              id="factCats1"
            >
```
Пофикси везде инпуты